frontend/src/api/Problem.ts

```diff
@@ -2,15 +2,17 @@ import axios from 'axios';
 import { ErrorResponse, errorHandler } from './Error';
 
 export type Problem = {
-  id: number,
+  problemId: string,
   name: string,
   description: string;
 }
 
 const basePath = '/api/v1/problems';
 const routes = {
   getProblems: `${basePath}/`,
-  postProblem: `${basePath}/`,
+  createProblem: `${basePath}/`,
+  getSingleProblem: (problemId: string) => `${basePath}/${problemId}`,
```

Comment:
When would this be used? Could you create a `getSingleRandomProblem` method that **doesn't** require a `problemId`, so that can be used instead of `getProblems`? How could you use the `getSingleRandomProblem` method to ensure that the same problem isn't returned twice in a row, if some users restart the game?

Reply:
I'm not sure about the part with ensuring that a problem isn't received twice in a row. Is this like on each page refresh or do you mean like if people play multiple games one after the other? 

Both situations seem tricky - might involve some localStorage stuff

Reply:
As for `getSingleProblem`, right now it's not used anywhere but I'm imagining once we get to the point where we start creating problems the endpoint might be used somewhat often for like an admin panel of some sorts (or like if users can create/edit their own problems in a kahoot-style game, although that's way in the future)

Reply:
I think storing in `localStorage` or perhaps in a group of user's accounts would be good, but that's definitely for the future, not necessary to change in this PR.

Thoughts on the `getSingleRandomProblem` method? I'll approve anyway since it's been too long since I've reviewed, but do you think you could implement that?